From 4eef1679b770fdf3e3f6209d68cd891ffb0aaa13 Mon Sep 17 00:00:00 2001
From: Olaf Freyer <unknown@examle.com>
Date: Wed, 30 Jul 2025 01:27:58 +0300
Subject: [PATCH 02/26] [ffmpeg] rename pp_mode_t and pp_context_t types

A patch from Gentoo to fix build against ffmpeg-0.10

Bug: https://bugs.gentoo.org/401287
Origin: https://github.com/gentoo/gentoo-historical-2/commit/225565efb23a36fb3552f4953f904449d49afbc7
Original-commiter: Samuli Suominen <ssuominen@gentoo.org>
Patch-name: transcode-1.1.7-ffmpeg-0.10.patch
Signed-off-by: Alexander Golubev <fatzer2@gmail.com>
---
 filter/filter_pp.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/filter/filter_pp.c b/filter/filter_pp.c
index 8790fd08..5b0a857a 100644
--- a/filter/filter_pp.c
+++ b/filter/filter_pp.c
@@ -38,8 +38,8 @@
 
 /* FIXME: these use the filter ID as an index--the ID can grow
  * arbitrarily large, so this needs to be fixed */
-static pp_mode_t *mode[100];
-static pp_context_t *context[100];
+static pp_mode *mode[100];
+static pp_context *context[100];
 static int width[100], height[100];
 static int pre[100];
 
-- 
2.49.1

